std/conv.d

```diff
@@ -3142,9 +3142,9 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source) && !is(Source == enum
             while (isDigit(i))
             {
                 sawDigits = true;        /* must have at least 1 digit   */
-                if (msdec < (0x7FFFFFFFFFFFL-10)/10)
```

Comment:
I'm unsure why this was made to max out at 47 bits, worry that it might overflow 64-bit reals, with their 53-bit mantissas, below? Tested this on Android/ARM with `long.max` assigned to a 64-bit real, no problem there.